### PR TITLE
4/7

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2023-04-07.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+| MAX(c1) |
++---------+
+| 1       |
++---------+
+
+```
+
+If you want to use my API key to play with it, just reply and I'll send you mine. I have a $20 monthly maximum so don't get any ideas!
+
+### Commit Graphs pn DoltHub
```
Comment (tbantle22) [inline emails/2023-04-07.md:63]:
```suggestion
### Commit Graphs on DoltHub
```

File: emails/2023-04-07.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+We launched [commit graphs on DoltHub](https://www.dolthub.com/blog/2023-04-03-introducing-the-dolthub-commit-graph/)! In this case, a picture is worth 1,000 words.
+
+[![DoltHub Commit Graphs](../images/commit-graph-dolthub.png)](https://www.dolthub.com/blog/2023-04-03-introducing-the-dolthub-commit-graph/)
+
+### Dolt and AWS
+
+Here we go again with the [Dolt replica](https://www.dolthub.com/blog/2023-04-05-versioned-mysql-replicas-on-hosted-dolt/) theme. If you're running MySQL in AWS, [Jason](https://www.dolthub.com/team#brian) explains how easy it is to get a [Hosted Dolt](https://hosted.doltdb.com/) replica going. [You should just do thi](https://www.dolthub.com/blog/2023-04-05-versioned-mysql-replicas-on-hosted-dolt/). Get your MySQL insurance now!
```
Comment (tbantle22) [inline emails/2023-04-07.md:71]:
```suggestion
Here we go again with the [Dolt replica](https://www.dolthub.com/blog/2023-04-05-versioned-mysql-replicas-on-hosted-dolt/) theme. If you're running MySQL in AWS, [Jason](https://www.dolthub.com/team#brian) explains how easy it is to get a [Hosted Dolt](https://hosted.doltdb.com/) replica going. [You should just do this](https://www.dolthub.com/blog/2023-04-05-versioned-mysql-replicas-on-hosted-dolt/). Get your MySQL insurance now!
```

**Review by tbantle22 — APPROVED** [review]

File: emails/2023-04-07.md (hunk 1 above)
Comment (bpf120) [inline emails/2023-04-07.md:63]:
```suggestion
### Commit Graphs on DoltHub
```

File: emails/2023-04-07.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+I know what you all are thinking, "Where was my Friday Tim email? I hope he's ok." 
+
+Well, I was on vacation last week so you get a Monday email instead of a Friday email. I appreciate all the concern even though I did not receive a single "Are you ok?" email. 
+
+### ChatGPT Integration
```
Comment (macneale4) [inline emails/2023-04-07.md:7]: I don't know if a hidden command line tool should be above the fold. Isn't AWS goodness going to be more likely to result in paying customers?

Comment (timsehn) [reply emails/2023-04-07.md:7]: I think it's just cooler.

**Pull request merged**